packages/libs/user-datasets/src/lib/Components/UploadForm.tsx

```diff
@@ -607,6 +607,7 @@ function validateForm<T extends string = string>(
       datasetType: datasetUploadType.type,
       projects: [projectId],
       dataUploadSelection,
+      visibility: 'private',
```

Comment:
Are we setting this as `private` during upload for simplicity, rather than allowing the user (in clinepi and mbio) to set visibility during the upload phase?

Reply:
It's partially for simplicity, and partially because I think people should review their user dataset before setting it to public.